### PR TITLE
Vagrantfile: add docker

### PR DESCRIPTION
**Pull request by proppy**

### CODE REVIEW

File: scripts/install_docker.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+
+curl -sSL https://get.docker.com/ubuntu/ | sh
```
Comment (mco-gh) [inline scripts/install_docker.sh:4]: Why not use sudo apt-get install docker.io (to avoid hardcoded path and pipe to shell)?

Comment (proppy) [inline scripts/install_docker.sh:4]: because that's an older version (1.3).

Comment (mco-gh) [inline scripts/install_docker.sh:4]: ok. When I run your command on a VBox, I get this error:

E: Unable to locate package lxc-docker

Looks like lxc-docker us a prereq.

Comment (mco-gh) [inline scripts/install_docker.sh:4]: Forget it, just realized I'm on a 32 bit ubuntu and I see you also modified the vagrantfile to use 64 bit. never mind, this LGTM.

Comment (proppy) [inline scripts/install_docker.sh:4]: It worked for me, when doing `vagrant up` from a blank VM.

Are you sure your vagrant arch is `amd64`?

File: Vagrantfile [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # you're doing.
 Vagrant.configure(2) do |config|
   # Use Ubuntu Trusty (14.x) 32-bit
```
Comment (mco-gh) [inline Vagrantfile:9]: Fix comment to reflect change to 64 bit ubuntu?

**Pull request merged**